code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -661,7 +661,7 @@
 		return TRUE
 	if(!O.reagents || !O.reagents.reagent_list.len) // other empty containers not accepted
 		return FALSE
-	if(istype(O, /obj/item/reagent_containers/syringe) || istype(O, /obj/item/reagent_containers/glass/bottle) || istype(O, /obj/item/reagent_containers/glass/beaker) || istype(O, /obj/item/reagent_containers/spray) || istype(O, /obj/item/reagent_containers/medspray))
+	if(istype(O, /obj/item/reagent_containers/syringe) || istype(O, /obj/item/reagent_containers/glass/bottle) || istype(O, /obj/item/reagent_containers/glass/beaker) || istype(O, /obj/item/reagent_containers/spray) || istype(O, /obj/item/reagent_containers/medspray) || istype(O, /obj/item/reagent_containers/gummy))
```

Comment:
why not just istype(O, /obj/item/reagent_containers)?

Reply:
because all foods and drinks are considered reagent_containers and this is supposed to be a chem fridge

Reply:
it should check if it's food and drinks and block that not a whitelist